hf-zip-readme.md

```diff
@@ -0,0 +1,29 @@
+This zip package includes a development build of Handsontable with the new formulas plugin.
```

Comment:
I'm guessing this file's no longer needed?

Reply:
How come? I still see an entry in our calendar on April 28th to send one more bundle, which this file will be needed in. Unless I don't know something?

Reply:
True, one more bundle is scheduled. And let's keep it a while longer, until we're sure the official release will happen